### DISCUSSION TITLE
University Life Review : Part 2 ft আমার মা | Nazia Shehnaz

### DISCUSSION THREAD

**Discussion by giscus** [Geek-a-Byte/geek-a-byte.github.io#1] [General]
# University Life Review : Part 2 ft আমার মা | Nazia Shehnaz

ছোটবেলায় আমাদের কে কিন্ডারগার্ডেনে আমাদের শিক্ষকরা আমাদের সবচেয়ে প্রিয় ব্যক্তিত্ব নিয়ে ৫ থেকে ১০ লাইনের প্যারাগ্রাফ লিখতে বলতেন। Most of the time, আমার সবচেয়ে প্রিয় আর কেও না, আমার মা-ই হতেন। প্যারাগ্রাফ এর শেষে খুব সুন্দর করে একটা এন্ডিং দিতাম, আমি আমার মাকে অনেক ভালবাসি। সময়ের সাথে আমরা যত matured হই, আমাদের চিন্তাধারা অনেকটাই পরিবর্তিত হয়। তাই আজকে সম্পূর্ণ fresh perspective থেকে “আমার মা” নিয়ে রচনা লিখব। অনেক বড় লিখা, তাই সময় না থাকলে skip করে যাবেন।

https://geek-a-byte.github.io/posts/ULR-part2/

**Comment by Geek-a-Byte** [comment]
testing my comment section :/
